mealie/services/scraper/cleaner.py

```diff
@@ -357,11 +363,18 @@ def clean_time(time_entry: str | timedelta | None) -> None | str:
                 return str(time_entry)
         case timedelta():
             return pretty_print_timedelta(time_entry)
+        case dict(time_entry):
+            if "minValue" in time_entry:
+                return clean_time(time_entry["minValue"])
+            return None
```

Comment:
```suggestion
        case { "minValue": str(value) }:
            return clean_time(value)
```

I think this is the right syntax for matching a key/value in a dict

Reply:
thx, also updated the list case.
should we allow strings only or also allow timedeltas?

Reply:
> should we allow strings only or also allow timedeltas?

That's interesting. I don't think that it would be likely we'd end up with a time delta as a property in the dict, so it's probably fine. But for the sake of completeness we could add it.

Reply:
apparently mypy does not like that too much. So leaving it out.